_posts/2020-11-06-JOIN-2020.md

```diff
@@ -0,0 +1,330 @@
+With an approach and a lot of good practice experiments done in other organizations.  
+
+### Session 2 - The power of Kata - Michaëla Broeckx
+
+<div style="position: relative; width: 100%; height: 0; padding-bottom: 55%;">
+    <iframe src="https://www.youtube.com/embed/2IUNBhErVD0" width="100%" height="100%;" style="position: absolute; left: 0; top: 0; bottom: 0; right: 0;"></iframe>
+</div>
+
+We will jointly look for efficient ways to convert good intentions into value, by investigating some personal challenges and how we succeeded or failed.
+We will examine these techniques and connect them to how our brain works.
+For every challenge we take on, our human tendency to take the easiest path (the one we imagine to be right) can be counterbalanced by deliberate scientific thinking.  
+
+### Session 3 - Failure Culture – what it means to fail and how we can gain value from it - Anke Maerz
+
+<div style="position: relative; width: 100%; height: 0; padding-bottom: 55%;">
+    <iframe src="https://www.youtube.com/embed/PayZ4lKsr-w" width="100%" height="100%;" style="position: absolute; left: 0; top: 0; bottom: 0; right: 0;"></iframe>
+</div>
+
+I don't like failing.
+I want to achieve a goal, I invest my time and energy, I am looking forward to achieving this goal, I am getting closer, closer - and fail.
+The goal is not achieved and I want to retreat into a dark corner and just forget.
+This is a perfectly normal human reaction.
+Yet, as we all know, no big endeavor happens without its fair share of failure - some huge achievements even happen out of failure alone...
+Hence, there is something useful to it.
+Something that even smells like success!
+This talk analyses different projects, their main impediments and success or failure factors, includes some interactive parts - and in the end (hopefully) leaves us with some renewed courage to face the failures in our lives... to gain value from them.  
+
+### Session 4 - Why don't you get my code? - Francis Laleman
+
+**Video under review - will be added soon!**
+
+This is a playful, interactive session on the concept of "code" - as a formal, grammar-and-syntax-driven substitute for "meaning".
+We all know that code is the central edifice of what it is that we are doing - at work, and, perhaps, even in life.
+But do we fully understand the profound relationship between code and meaning? Do we really grasp how code, in all its formality and conventions, both pins us down into the confinement of "knowns", and has the power to liberate us from what is already known and drive us forward?
+After this session, expect to be aware of more than just "code", and to be able to understand "meaning", in life and work, through a variety of grammars and syntaxes, more diverse than ever imagined.  
+
+## In the spotlight - Ghelamco alert
+
+This talk features the awesome Internship project by Kevin Govearts which is a shining example of and end-to-end integrated IoT project.
```

Comment:
I would rephrase it as followed because "this talk" gives the appearance that it refers to the talk above.
```suggestion
The awesome internship project by Kevin Govaerts is a shining example of an end-to-end integrated IoT project.
```